package.json

```diff
@@ -92,6 +92,7 @@
   "dependencies": {
     "bower": "^1.7.9",
     "classnames": "^2.2.5",
+    "moment": "^2.15.0",
```

Comment:
I believe this should be discussed. I see that datepicker depends heavily on moment however in many casese this dependency can be avoided.